package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "prepare": "husky install",
     "build": "turbo run build",
-    "build-vscode": "turbo run build --filter=neo4j-for-vscode",
+    "build-vscode": "turbo run build neo4j-for-vscode#build:dev --filter !neo4j-for-vscode",
```

Comment:
perhaps there's a better way to do this.. 

because we're dependent on a build to run the extension, and turbo runs tasks in parallel, I found that there was a race condition when both the build and build:dev tasks were running and one would overwrite the other

so here I am filtering out the vscode extension package from the build task, and running the new vscode build:dev task to ensure that only build:dev is ran on the vscode extension

Reply:
I'm not understanding how both `build` and `build#dev` tasks end up being run in parallel?

Reply:
alright, I've taken another look at this - I've reverted this back to simply `turbo run neo4j-for-vscode#build:dev` 

when I first had this configuration in, both the build and the build:dev tasks were running

now only the build:dev task is running as I'd expect.. I don't _really_ understand what changed, but this looks much better